examples/CMakeLists.txt

```diff
@@ -4,11 +4,5 @@ file(GLOB files "*.cpp")
 foreach(file ${files})
     get_filename_component(file_basename ${file} NAME_WE)
     add_executable(${file_basename} ${file})
-    set_target_properties(${file_basename} PROPERTIES
-            CXX_STANDARD 14
-            CXX_STANDARD_REQUIRED ON
-    )
-    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -stdlib=libc++ -lc++abi")
```

Comment:
The `-stdlib=libc++` gave me compiler errors since it was not understood by my compiler, not sure what it is trying to accomplish?

Reply:
I think you better talk to previous cmake commiter cause I know nothing about it - I don't use cmake and merged his PR just because some people asked about it

Reply:
Good idea, I was looking at the commit history for that file and found you with https://github.com/fnc12/sqlite_orm/commit/b0dca907355013e5d9580fba19306da4dfba036b but the rest of the file is written by @DerShokus . I assume it's them I should ask. So, @DerShokus what is your opinion on the matter? Thanks

Reply:
yes, @DerShokus created cmake scripts. Also please change the target branch to `dev` and I'll merge the PR. Thanks. Your problem with iterators appeared because my CI service runs only on mac. I set it up like this cause ubuntu is too old for C++14 there. It is better to have CI for win, mac, ios, android, linux, orange pi. I'm thinking about this